src/Actions/Actions/CaptureAction.cs

```diff
@@ -27,7 +26,6 @@ public static class CaptureAction
         // Test hooks; making this non-static and enabling constructor injection
         // would be nicer, but this was the minimal set of changes to remove the
         // need to use Fakes to test this class.
-        internal static Func<DataManager> GetDataManager = () => DataManager.GetDefaultInstance();
```

Comment:
This hook existed for testing. It's no longer needed since we can control it via IActionContext.DataManager